src/components/Pages/Unit/widgets/UnitMaterials/Transcription/Transcription.js

```diff
@@ -112,6 +131,10 @@ class Transcription extends Component {
     if (selectedFile !== prevState.selectedFile || language !== prevState.language) {
       this.loadFile(selectedFile);
     }
+
+    if (selectedFile && this.props.doc2htmlById[selectedFile.id] && this.props.doc2htmlById[selectedFile.id].wip === false) {
+      scrollToSearch();
```

Comment:
This will be executed on each re-render after the first successful one

Reply:
fixe